app/src/main/java/org/mozilla/fenix/tabstray/browser/NormalBrowserTrayList.kt

```diff
@@ -41,34 +44,41 @@ class NormalBrowserTrayList @JvmOverloads constructor(
 
     override val tabsFeature by lazy {
         val tabsAdapter = concatAdapter.browserAdapter
+        val inactiveTabsEnabled = context.settings().inactiveTabsAreEnabled
+        val tabFilter: (TabSessionState) -> Boolean = {
+            when {
+                FeatureFlags.tabGroupFeature && inactiveTabsEnabled ->
```

Comment:
Do we need to check the feature flag if `inactiveTabsEnabled` is now a `featureFlagPreference`?

Reply:
This is the FeatureFlag for tab groups not inactive tabs

Reply:
Ah, forgot about that.  Then this make sense.  Thanks